hack/validate-release-jobs.py

```diff
@@ -14,13 +14,23 @@
 job_definitions_path = 'ci-operator/jobs/openshift/release'
 
 
+def raise_on_duplicates(ordered_pairs):
+    d = {}
+    for k, v in ordered_pairs:
+        if k in d:
+            raise ValueError("Duplicate key: %r for value: %r" % (k, v))
+        else:
```

Comment:
nit: no need for this `else`, because raising the error will have already broken the loop.

```python
for k, v in ordered_pairs:
    if k in d:
        raise ValueError("Duplicate key: %r for value: %r" % (k, v))
    d[k] = v